web/src/store/index.js

```diff
@@ -76,8 +99,8 @@ function _invalidatePlots(state, { key, plotList }) {
     Vue.set(state.plots[key][name], 'valid', false);
   });
 }
+
 function _setLables(state, { key, rows, columns }) {
-  _invalidatePlots(state, { key, plotList: ['pca', 'loadings'] });
```

Comment:
why is the plot invalidation not longer needed?

Reply:
Because right now changing a label always triggers load_dataset which itself always invalidates plots.  Doing it twice was causing the plots to load before and after.  I plan on refactoring this when I make the `validation` changes mentioned in slack today.